### PR TITLE
[15.0] [MIG] fleet_vehicle_service_kanban: Migration to 15.0

### PR DESCRIPTION
**Pull request by rodrigonevest**

### CODE REVIEW

**Review by marcelsavegnago — APPROVED** [review]

**Comment by marcelsavegnago** [comment]
ping @ivantodorovich

File: fleet_vehicle_service_kanban/tests/test_fleet_vehicle_log_service.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            }
+        )
+
+        self.service_type = self.env["fleet.service.type"].create(
+            {"name": "Fatura do Fornecedor", "category": "service"}
+        )
+
+        # self.purchase = self.env["purchase.order"].create({
+        #     "name":"New"
+        # })
```
Comment (ivantodorovich) [inline fleet_vehicle_service_kanban/tests/test_fleet_vehicle_log_service.py]: Could you remove these commented lines 🙏🏻 ?

**Review by ivantodorovich — APPROVED** [review]

File: fleet_vehicle_service_kanban/tests/test_fleet_vehicle_log_service.py (hunk 1 above)
Comment (rodrigonevest) [reply fleet_vehicle_service_kanban/tests/test_fleet_vehicle_log_service.py]: @ivantodorovich sorry for the late reply, done.

**Pull request merged**